### ISSUE TITLE
Default `waitImage.repository` is not compatible with `wait.sh` script

### ISSUE THREAD

**Issue by amilstead** [Kong/charts#201]
With a completely vanilla install (with or without a bundled postgres) the migration init scripts fail with: 

```
Error: failed to start container "wait-for-postgres": Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"bash\": executable file not found in $PATH": unknown
```

This is because [`wait.sh`](https://github.com/Kong/charts/blob/main/charts/kong/templates/wait-for-postgres-script.yaml#L8) uses `until timeout 2 bash`, and the default `waitImage` repo is `busybox`, which definitely does not have `bash`.

**Comment by rainest** [comment]
@amilstead can you clarify what you mean by:
> With a completely vanilla install 

That default image [did change in the the 1.9 release](https://github.com/Kong/charts/blob/main/charts/kong/UPGRADE.md#changes-to-wait-for-postgres-image), but the current default values.yaml reflects that:
https://github.com/Kong/charts/blob/ac20c1e6ff1b81ab408d570964f70b7af9518f81/charts/kong/values.yaml#L397-L400

If you have a values.yaml copied from an older release, the old default image configuration in it will result in the error you saw. You'll most likely want to remove the `waitImage` section from your values.yaml if so (it's usually unnecessary to include it--Helm will use the current default if it's absent, and you should only need to explicitly configure it if you use a custom image or registry).

**Comment by amilstead** [comment]
> @amilstead can you clarify what you mean by:
>> With a completely vanilla install

I used a fresh helm repo add/helm upgrade --install with a `values.yaml` that I pulled from this repository.

> but the current default values.yaml reflects that:

That's gotta be it!

I could have sworn I pulled down the most recent raw of `values.yaml` but obviously not, as it has very clearly been `bash` for almost three weeks. 

I think what happened here was: I got a version of the helm chart that uses the `bash`-based `wait.sh`, but a `values.yaml` from a version of the release prior to the helm chart I was on.

My mistake!
